### ISSUE TITLE
keybase filesystem, unable to create a folder

### ISSUE THREAD

**Issue by matthewp** [keybase/client#2069]
Keybase GUI Version: 1.0.13-20160218170011+513e03d

<img width="504" alt="screen shot 2016-02-19 at 11 10 38 am" src="https://cloud.githubusercontent.com/assets/361671/13181115/73a88e70-d6f9-11e5-9f83-a65e762dc481.png">

**Comment by matthewp** [comment]
input/output errors when trying to copy a file into the folder as well.

**Comment by maxtaco** [comment]
Can you `keybase log send`? Thanks!

**Comment by matthewp** [comment]
Just did, log id is dc6e8c969ba4bbbc3321631c

**Comment by strib** [comment]
Hi @matthewp, thanks for the report.  According to the logs, Keybase tried to pop up a pinentry dialog for you, to enter your password and unlock your device private key, to allow you to establish a session with our file servers. But it looks like you hit "cancel" on that pinentry, and the cancel has been cached by the service for a short period of time so it doesn't bother you with more popups (10 minutes I think?).

If you don't want to wait, you can manually run `keybase unlock` to get another pinentry popup.

If you never saw a popup, let us know -- it's possible your GUI isn't working in that case.

**Comment by matthewp** [comment]
I did not get a popup when trying to copy a file into the folder. I did after unlocking. This is roughly what happened:

I installed the osx app, did `keybase login`... I got a passphrase popup but I can't exactly remember what prompted it.  I did hit cancel after entering wrong passphrases several times. I ran `keybase passphrase change` and changed the passphrase and relogged in.  After all of this I tried copying files over (from the command line and from the finder) and got the errors listed.

Somewhat complex scenario so probably not helpful to you guys.
